packages/framework-core/src/booster-rocket-dispatcher.ts

```diff
@@ -3,7 +3,7 @@ import { BoosterConfig, rocketFunctionIDEnvVar } from '@boostercloud/framework-t
 export class BoosterRocketDispatcher {
   constructor(readonly config: BoosterConfig) {}
 
-  public dispatch(request: unknown): Promise<unknown> {
+  public async dispatch(request: unknown): Promise<unknown> {
```

Comment:
🤔 If I have seen everything correctly, this change is only adding the async keyword, but it is now needed, as the code inside the function doesn't have any `await`. 

You are already returning a promise so, unless you want to use await inside the function, you don't need to add async. 

Was this because of some bug?

Reply:
You are right, it's not needed